gensim/corpora/mmcorpus.py

```diff
@@ -19,21 +19,78 @@
 logger = logging.getLogger('gensim.corpora.mmcorpus')
 
 
-class MmCorpus(matutils.MmReader, IndexedCorpus):
+try:
+    # try to load fast, cythonized code if possible
+    from gensim.corpora._mmreader import MmReader
+    FAST_VERSION = 1
```

Comment:
No need `FAST_VERSION` here at all (this is connected with BLAS stuff, that isn't used here).

Reply:
sounds good will remove this too